src/components/reviews/reviewTile.jsx

```diff
@@ -96,11 +98,17 @@ class ReviewTile extends React.Component {
     let { review } = this.props;
     return (
       <div className="tile is-child box">
-        <div>{review.rating}-star rating goes here</div>
-        {this.state.verified
-          ? <div className="has-text-right is-size-7">{review.reviewer_name}, {moment(review.date).format("MMMM DD, YYYY")}
-            <br />&#10004; Verified user</div>
-          : <div className="has-text-right is-size-7">{review.reviewer_name}, {moment(review.date).format("MMMM DD, YYYY")}</div>}
+        <div className="level">
+          <div className="level-left">
+            <div className="level-item"><StarRating rating={review.rating} width="15" height="15" /></div>
+          </div>
+          <div className="level-right">
+            {this.state.verified
+              ? <div className="level-item is-size-7">{review.reviewer_name}, {moment(review.date).format("MMMM DD, YYYY")}
```

Comment:
so this is the "moment" you mentioned. Wish we had gotten this working before i wrote that function of my end 😂